q2_quality_control/plugin_setup.py

```diff
@@ -46,9 +46,13 @@
         'method': ('Alignment method to use for matching feature sequences '
                    'against reference sequences'),
         'perc_identity': ('Reject match if percent identity to reference is '
-                          'lower. Must be in range [0.0, 1.0]'),
-        'evalue': 'BLAST expectation value (E) threshold for saving hits.',
-        'threads': 'Number of jobs to execute if method == vsearch',
+                          'lower. (or if E value is higher than E value '
+                          'threshold.)  Must be in range [0.0, 1.0]'),
+        'evalue': ('BLAST expectation (E) value threshold for saving hits. '
```

Comment:
Since you noticed that it was very hard to actually get sequence to fail to hit based on it's E value, would it make more sense to make that an optional parameter, with a default of `None`? That way users can experiment with it if they want to, but it otherwise would be guaranteed to not impact the results? I would also still be ok with just dropping it all together, now that you've added `perc_query_aligned`.

Reply:
👍 
Setting to `None` will cause an error in the blast subprocess. Instead, I have this set to the same default used in blastn.

If you prefer, I can optionally append evalue to the subprocess only if it is set with something like:
```
if evalue is None:
    cmd.append(['--evalue', evalue])
```

and that way I do not need to imitate blast defaults. Let me know if you want this change.

Reply:
I think appending is a better way to handle it (i.e., the example you added here). The reason to prefer that approach is that if the `blastn` default changes, you wouldn't need to update your code.

Reply:
👍

Reply:
Though note that that should be: `if evalue is not None`